src/index/visitor.rs

```diff
@@ -252,6 +252,14 @@ fn visit_data_type(
                 information,
                 nature: TypeNature::Derived,
             });
+            let global_struct_name = format!("{}__init", name);
```

Comment:
what are we ever doing with this new global?
is there something still missing? probably this is still WIP?

Reply:
Yes this is a draft
The idea is to memcpy the struct on init, similar to what C would do.